hrjobcontract/js/build.js

```diff
@@ -9,7 +9,7 @@
     'common': 'empty:',
     'job-contract/vendor/fraction': 'job-contract/vendor/fraction',
     'job-contract/vendor/job-summary': 'job-contract/vendor/jobsummary',
-    'leave-absences': '%{uk.co.compucorp.civicrm.hrleaveandabsences}/js/angular/src/leave-absences'
+    'leave-absences': '../../../uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences'
```

Comment:
why? we should not use relative paths at all. i'm surprised that this significant change was not even mentioned in the PR, we did a lot of work to be able to achieve cross-extension import capability, and now we're getting rid of it?

Reply:
and you figured out why?

Reply:
done. the problem was that it called old require-js func that is not needed anymore. also watch func was old.